lang/interpret_test/TestAstFunc2/escaping1/main.mcl

```diff
@@ -46,23 +46,17 @@ test "S: \\$ EOF" {}
 
 test "T: newline\nEOF" {}
 
-# XXX: possible bugs or misunderstood expectations:
-
-#test "W: \\$" {}
-# got: <error>
-# exp: W: \$
-
-#$x1 = "This is x1"
-#test "X: $${x1} EOF" {}
-# got: X: $${x1} EOF
-# exp: X: $This is x1 EOF
-
-#$unused = "i am unused"
-#$y1 = "{unused}"
-#test "Y: $${y1} EOF" {} # check there isn't double parsing
-# got: Y: $${y1} EOF
-# exp: Y: ${unused} EOF
-
-#test "Z: $$$" {}
-# got: <error>
-# exp: Z: $$$ EOF
+test "U: tab\t\\\ttabEOF" {}
+
+# test "V: " {}
```

Comment:
Should this be commented?

Reply:
Yeah, I couldn't come up with another test 😛 Decided to leave the empty case instead of defacing the alphabet